### PR TITLE
Epic 7 -Dmytro Sydor

### PR DESCRIPTION
**Pull request by dmytrosydor**

### CODE REVIEW

**Comment by VoroninOlexandr** [comment]
Screenshot_19

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/dmytro_sydor/Epic 7/control.cpp [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    const string intSumFileName = "sumOutputInt.txt";
+    const string floatSumFileName = "sumOutputFloat.txt";
+
+    // Обчислення та виведення добутку елементів з файлів
+    float intProduct = calculateProductFromFile(intSumFileName);
+    float floatProduct = calculateProductFromFile(floatSumFileName);
+
+    cout << "Product of integers in the file: " << intProduct << endl;
+    cout << "Product of floats in the file: " << floatProduct << endl;
+
```
Comment (fm2lpnu) [inline ai_12/dmytro_sydor/Epic 7/control.cpp:76]: Task 6 is not implemented.

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**